src/pages/index.js

```diff
@@ -72,58 +265,35 @@ function handleAddCardSubmitForm() {
 	const name = utils.newCardTitleInput.value; //newCardTitleValue
 	const link = utils.newCardUrlInput.value; //newCardUrlValue
```

Comment:
Only method `_getInputValues` collects all the data of the inputs **inside** `PopupWithForm` and passes it to the call of `submitForm`. So you need to use the collected data rather than the inputs.

**Please, correct it everywhere**